MapboxNavigation/NavigationViewController.swift

```diff
@@ -165,15 +166,6 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
     public var routeController: RouteController!
     
     /**
-     `simulate` provides simulated location updates along the given route.
-     */
-    public var simulatesLocationUpdates: Bool = false {
```

Comment:
Should we keep this bool for simplicity in the Navigation UI and have the didSet function optionally set the SimulatedLocationManager?